front/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "micronation",
-  "version": "2.3.2",
+  "version": "2.3.3",
```

Comment:
We should have a major release from here

Reply:
Nevermind, I thought this was the backend, and tbh this would usually be done on the main branch instead of in a PR.
Here, it was done for deploying a testing version in preprod of this branch, which is also a valid use case imho.